cmd/root.go

```diff
@@ -216,6 +216,9 @@ func initConfig() {
 	viper.BindEnv("TRACING_SERVICE_NAME")
 	viper.SetDefault("TRACING_SERVICE_NAME", "Ory Hydra")
 
+	viper.BindEnv("TRACING_OMIT_SQL_ARGUMENTS_FROM_SPANS")
```

Comment:
Please document this around here: https://github.com/ory/hydra/blob/master/cmd/serve.go#L182

And while you're add it, please document the other tracing env vars there too :)